collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -116,7 +117,7 @@
     private SimpleAdapter mFormListAdapter;
     private ArrayList<HashMap<String, String>> mFormList;
 
-    private boolean mToggled = false;
+    // private boolean mToggled = false;
```

Comment:
We get history through git so I think code that's no longer needed should be deleted.